gen/dcos-config.yaml

```diff
@@ -448,7 +448,7 @@ package:
       MESOS_REGISTRY_STORE_TIMEOUT=60secs
       MESOS_REGISTRY_FETCH_TIMEOUT=60secs
       MESOS_REGISTRY_STRICT=false
-      MESOS_SLAVE_REMOVAL_RATE_LIMIT=1/20mins
+      MESOS_SLAVE_REMOVAL_RATE_LIMIT=100/1mins
```

Comment:
Do you think that maybe we should just set the rate to a faster interval?

1/15secs?

Reply:
anything is possible... I don't think it should exist actually... 
Also... I have a strong preference to NOT have what you are suggesting...  any value with a low numerator has the effect of causing seemingly unexplained delays of propagation of information... lets say you lost 100 nodes at once... with a 1/15 secs every 15 secs you would get a new notification of lost node until all 100 are reported... which is 4 each min... it would take 25 mins before you got the last notification.     Mesos is a 2 level scheduler.   The 2nd level is the intelligent layer.  we should not have mesos hide information from the intelligent layer.